perf.groovy

```diff
@@ -578,9 +568,9 @@ parallel(
 
             if (isPR) {
                 TriggerBuilder builder = TriggerBuilder.triggerOnPullRequest()
-                builder.setGithubContext("${os} ${arch} ILLink Perf Tests")
+                builder.setGithubContext("${os} ${arch} Performance Scenarios Tests")
                 builder.triggerOnlyOnComment()
-                builder.setCustomTriggerPhrase("(?i).*test\\W+${os}\\W+${arch}\\W+illink\\W+perf.*")
+                builder.setCustomTriggerPhrase("(?i).*test\\W+${os}\\W+${arch}\\W+scenarios.*")
```

Comment:
I'd like the trigger to still have perf in it, so that we know we are running perf testing.